DynMiner2/cStatDisplay.cpp

```diff
@@ -74,6 +75,40 @@ void cStatDisplay::displayStats(cSubmitter *submitter, string mode, int hiveos)
 
             std::string uptime = seconds_to_uptime(difftime(now, start));
 
+	    if (!statURL.empty()) {
+
+	        curl = curl_easy_init();
+	        curl_easy_setopt(curl, CURLOPT_URL, statURL.c_str());
+
+	        string hashStr = to_string(int(hashrate));
+
+	        char postString[320];
+
+	        sprintf(postString, R"(
+	                    {
+			     "Name": "%s",
+			     "Hashrate": %s,
+			     "Submit": %4lu,
+			     "Accept": %-4d,
+			     "Reject": %4d,
+			     "Diff": %-4d
+			    }
+					  )",
+			minerName.c_str(), hashStr.c_str(),
+                totalStats->share_count.load(std::memory_order_relaxed), totalStats->accepted_share_count.load(std::memory_order_relaxed),
+                totalStats->rejected_share_count.load(std::memory_order_relaxed), totalStats->latest_diff.load(std::memory_order_relaxed));
+
+		curl_easy_setopt(curl, CURLOPT_POSTFIELDS, postString);
+
+		res = curl_easy_perform(curl);
```

Comment:
Same comment as above - this should probably be a local var, especially as it's literally used here and nowhere else.

Reply:
Fixed as well per your suggestion.